torch/compiler/__init__.py

```diff
@@ -92,3 +93,9 @@ def disable(fn=None, recursive=True):
     import torch._dynamo
 
     return torch._dynamo.disable(fn, recursive)
+
+def cudagraph_mark_step_begin():
+    "Indicates that a new iteration of inference or training is about to begin."
```

Comment:
For the docstring to be useful it should instead say when do you expect people to call this and why?

A short example also would be nice